src/app/component/user/auth/sign-in/sign-in.component.css

```diff
@@ -1,17 +1,312 @@
+.grey-bg {
+  width: 100%;
+  height: 100vw;
+  background: #292D35;
+  padding: 10px 0;
+}
+
 .sign-in {
-  display: block;
-  position: absolute;
-  top: 50%;
-  left: 50%;
-  transform: translateX(-50%) translateY(-50%);
-  width: 30%;
+  width: 50%;
+  height: auto;
+}
+
+.outher-wrapper {
+  width: 100%;
+  height: 100%;
 }
 
 .sign-in input {
+  display: block;
+  width: 100%;
+  height: 25px;
+  border-radius: 5px;
+  -webkit-border-radius: 5px;
+  -moz-border-radius: 5px;
+  -o-border-radius: 5px;
+  border: 1px solid #839c94;
+  background-color: rgba(131, 156, 148, 0.07);
+  padding: 0 10px;
+}
+
+label[for="email"],
+label[for="password"],
+label[for="first-name"] {
+  font: 16px 'OpenSans', sans-serif;
+  letter-spacing: 0.34px;
+  color: #839c94;
+  margin-top: 10px;
+}
+
+.error,
+.validation-error {
+  color: #f03127;
+  font-size: 14px;
+}
+
+.wrapper-container {
+  display: block;
+  margin: 0 auto !important;
+  max-width: 800px;
+  width: 65%;
+  
+  z-index: 2;
+  background: #ffffff;
+}
+
+.form-wrapper {
+  display: flex;
+  justify-content: space-between;
+  max-width: 800px;
+}
+
+.form-pic {
+  width: 340px;
+  height: auto;
+}
+
+.form-pic>img {
+  width: 100%;
+  height: auto;
+}
+
+form>h1 {
+  font: 30px 'OpenSans-Bold', sans-serif;
+  letter-spacing: 0.18px;
+  font-weight: 600;
+  margin: 10% 0;
+}
+
+form>h2 {
+  font: 18px 'OpenSans-Regular', sans-serif;
+  line-height: 22px;
+  letter-spacing: 0.18px;
+  font-weight: 500;
+  margin-top: 5px;
+}
+
+form {
+  margin: 2% 5% 5% 5%;
+}
+
+.red {
+  border: 1px solid #f03127 !important;
+  background: #FEF2F1;
+}
+
+.grey {
+  border: 1px solid #839c94;
+}
+
+.img {
+  cursor: pointer;
+  width: 24px;
+  height: 20px;
+  float: right;
+  margin-right: 10px;
+  margin-top: -22px;
```

Comment:
avoid negative values

Reply:
in this case we don't use Bootstrap but bootstrap is still in the project and styles from it's are in conflict with pure css..... it will be possible to fix after removing bootstrap from the project